tests/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ add_subdirectory(str)
 if (OE_SGX)
 add_subdirectory(aesm)
 add_subdirectory(debugger)
+add_subdirectory(report)
```

Comment:
Oh, this should probably be in the `if(ADD_WINDOWS_ENCLAVE_TESTS)` statement since it creates/uses enclaves.

Reply:
Good call, fixed